SimpleDrive.App/ClientApp/services/authentication.service.ts

```diff
@@ -0,0 +1,45 @@
+import http from "./core/http";
+import { AxiosResponse } from "axios";
+
+import { LoginInfo } from "../models/login-model";
+import { UserProfile } from "../models/user-profile";
+
+class AuthenticationService {
+    private static currentUserKey = 'currentUser';
```

Comment:
`public static readonly`
`CURRENT_USER_KEY`

Reply:
Fixed in https://github.com/SimpleDriveOrganization/SimpleDrive/pull/14/commits/ed82b6545cfea2abe971dd85811ed9519b10e325